slim/html5/block_audio.html.slim

```diff
@@ -1,6 +1,8 @@
-.audioblock id=@id class=[@style,role]
-  - if title?
-    .title=captioned_title
+= block_with_captitle ['audioblock', style]
```

Comment:
I’ve found that it’s actually unnecessary to explicitly distinct between captioned and uncaptioned titles — method `captioned_title` is [defined in AbstractBlock](https://github.com/asciidoctor/asciidoctor/blob/4d0ae51641c773004355879f5642eb18b6046dce/lib/asciidoctor/abstract_block.rb#L142-L146) and it returns just naked title when the nodes doesn’t have a caption.

So problem solved, we don’t need extra helper for captioned titles at all.